benchmarks/transport/ghex_p2p_bi_cb_avail_mt.cpp

```diff
@@ -79,15 +79,7 @@ int main(int argc, char *argv[])
     }
 #endif
 
-#ifdef USE_OPENMP
```

Comment:
Why are these removed? Is the benchmark explicitly not using OpenMP? (also for the other similar changes)

Reply:
This has been simply moved and cleaned up. I use the primitives, and the OMP check is moved to the front of the file.